apps/backend/src/db/migrations/20240818081459_add_user_details_table.ts

```diff
@@ -0,0 +1,37 @@
+import type { Knex } from "knex";
```

Comment:
```suggestion
import { type Knex } from "knex";
```
To have the same approach everywhere